src/main/java/org/deepsymmetry/beatlink/data/OpusProvider.java

```diff
@@ -86,7 +114,6 @@ public boolean isRunning() {
      */
     @API(status = API.Status.STABLE)
     public synchronized void attachMetadataArchive(File archive, CdjStatus.TrackSourceSlot slot) throws IOException {
```

Comment:
We’ll want to change this API so that we are attaching archives to device numbers (1-3). We will assume the slot is the USB slot.

Reply:
(this can wait until a later PR)